src/main/java/org/kizombadev/markdownparser/Application.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Marcel Swoboda
+ * Copyright (C) 2017
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ */
+
+package org.kizombadev.markdownparser;
+
+import org.apache.commons.cli.*;
+import org.apache.log4j.Logger;
+import org.kizombadev.markdownparser.exceptions.MarkdownParserException;
+
+import java.io.*;
+
+public class Application {
+    private static final String OUTPUT_OPTION = "output";
+    private static final String INPUT_OPTION = "input";
+    private static final String HELP_OPTION = "help";
+    private final static Logger log = Logger.getLogger(Application.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)